weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareEvaluationsPage/CompareEvaluationsPage.tsx

```diff
@@ -237,3 +241,49 @@ const ResultExplorer: React.FC<{state: EvaluationComparisonState}> = ({
     </VerticalBox>
   );
 };
+
+const InvalidEvaluationBanner: React.FC<{state: EvaluationComparisonState}> = ({
+  state,
+}) => {
+  const [dismissed, setDismissed] = useState(false);
+  const invalidEvals = useMemo(() => {
+    return Object.values(state.data.evaluationCalls)
+      .filter(evalCall => {
+        return Object.keys(evalCall.summaryMetrics).length === 0;
+      })
+      .map(call =>
+        call.name !== EVALUATION_NAME_DEFAULT
+          ? call.name
+          : call.callId.slice(-4)
+      );
+  }, [state.data.evaluationCalls]);
+  if (invalidEvals.length === 0 || dismissed) {
+    return null;
+  }
+  return (
+    <Box
+      sx={{
+        width: '100%',
+        paddingLeft: STANDARD_PADDING,
+        paddingRight: STANDARD_PADDING,
+      }}>
+      <Alert
+        severity="info"
+        style={{backgroundColor: hexToRGB(TEAL_300, 0.3), color: '#038194'}}
```

Comment:
We shouldn't be using hexcodes for colors, I think this should be teal-600. As a nit there's an effort in general to move towards using tailwind for this sort of styling.